spec/index.html

```diff
@@ -1140,7 +1199,10 @@ <h2>RDF/XML Internet Media Type, File Extension, and Macintosh File Type</h2>
       systems be given a file type of <code>"rdf&nbsp;"</code>
       (all lowercase, with a space character as the fourth letter).</p>
 
-      <p>The `application/rdf+xml` media type has been registered at IANA as [[RFC3870]].</p>
+    <p>Features introduced in RDF 1.2 require a version announcement in the
```

Comment:
That's an independent decision?

I think "required" is a burden because to stream the whole data must be seen. Therefore, I think the version should be optional.

The majority of RDF 1.2 data will be legal RDF 1.1 .

Reply:
This is just the PR which introduces versioning.

Reply:
The wording says that using 1.2 features requires a version announcement, not that any use of RDF/XML requires a version announcement.